apps/framework-cli/src/utilities/system.rs

```diff
@@ -13,3 +13,24 @@ pub fn copy_directory(source: &PathBuf, destination: &PathBuf) -> Result<(), std
 
     Ok(())
 }
+
+pub fn read_directory<P: AsRef<Path>>(path: P) -> io::Result<Vec<String>> {
```

Comment:
Not sure if you looked at the standard library, if so why not using it? 

if not:
https://doc.rust-lang.org/std/fs/fn.read_dir.html

Same for the copy command